### PR TITLE
Correction of showOpenDialog without the expected return for a promise

### PR DESCRIPTION
**Pull request by daniielamaro**
Without this change, it is impossible to change the default download folder due to not having the appropriate return for a promise

### CODE REVIEW

File: src/preload/index.d.ts
Comment (zamitto) [inline src/preload/index.d.ts:1]: the changes on this file are not needed, if you check the return type it i already a Promise

File: src/preload/index.ts
Comment (zamitto) [inline src/preload/index.ts:1]: the changes on this file are not needed, if you check the return type it i already a Promise

File: src/preload/index.d.ts
Comment (daniielamaro) [reply src/preload/index.d.ts:1]: Yes, it is a promise, but the objects that are using this promise are not using the appropriate return for a promise, if you have any doubts, do a test and try changing your download folder in settings

File: src/preload/index.ts
Comment (daniielamaro) [reply src/preload/index.ts:1]: Yes, it is a promise, but the objects that are using this promise are not using the appropriate return for a promise, if you have any doubts, do a test and try changing your download folder in settings

File: src/preload/index.d.ts
Comment (zamitto) [reply src/preload/index.d.ts:1]: yes, but the changes on the index.d.ts and index.ts files are not needed. The changes on show-open-dialog.ts are the changes that fix the problem. 
I just tested it

Comment (daniielamaro) [reply src/preload/index.d.ts:1]: Você está certo, enviei mais mudanças do que o necessario, já corrigi e deixei so o necessário para corrigir o problema

Comment (daniielamaro) [reply src/preload/index.d.ts:1]: You're right, I sent more changes than necessary, I already corrected them and left only what was necessary to fix the problem

**Comment by zamitto** [comment]
This is fixed now on main branch. Thank you for your collaboration anyways

**Pull request closed without merging**